### PR TITLE
i18n(ko-KR): add `cloudflare.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- add `cloudflare.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+DB_PASSWORD=myPassword
+```
+
+cli 명령에 `wrangler`를 사용하려는 경우 (예를 들어, D1 마이그레이션) Astro 프로젝트 루트에 올바른 콘텐츠가 작성된 `wrangler.toml` 파일을 추가해야 합니다. 자세한 내용은 [Cloudflare 문서](https://developers.cloudflare.com/)를 참조하세요.
+
+```toml title="wrangler.toml"
+name = "example"
+compatibility_date = "2023-06-14"
+
+# example for D1 Binding
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/cloudflare.mdx:360]: Can you translate this comment?

File: src/content/docs/ko/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+```
+
+#### Cloudflare Workers
+
+workers 프로젝트를 프로덕션에 배포하려면 Astro 프로젝트의 루트 디렉터리에 있는 `wrangler.toml` 구성 파일을 사용하여 바인딩을 구성해야 합니다. 바인딩에 로컬로 액세스할 수 있도록 `@astrojs/cloudflare` 어댑터는 `wrangler.toml` 파일도 읽습니다.
+
+```toml title="wrangler.toml"
+name = "example"
+
+# example of a KV Binding
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/cloudflare.mdx:375]: This one as well.

File: src/content/docs/ko/guides/integrations-guide/cloudflare.mdx [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+kv_namespaces = [
+    { binding = "KV", id = "KV", preview_id = "KV" },
+]
+
+# example of a var binding (environment variables)
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/cloudflare.mdx:380]: Also this one.

File: src/content/docs/ko/guides/integrations-guide/cloudflare.mdx [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+[vars]
+URL = "example.com"
+
+# example of a D1 Binding
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/cloudflare.mdx:384]: Also this one.

File: src/content/docs/ko/guides/integrations-guide/cloudflare.mdx [hunk 5]
```diff
@@ -0,0 +1,571 @@
...
+[[d1_databases]]
+binding = "D1"
+database_name = "D1"
+database_id = "D1"
+preview_database_id = "D1"
+
+# example of a R2 Binding
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/cloudflare.mdx:391]: Also this one.

File: src/content/docs/ko/guides/integrations-guide/cloudflare.mdx [hunk 6]
```diff
@@ -0,0 +1,571 @@
...
+[[r2_buckets]]
+binding = 'R2'
+bucket_name = 'R2'
+
+# example of a Durable Object Binding
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/cloudflare.mdx:396]: Also this one.

**Review by dreyfus92 — COMMENTED** [review]
Hey @jsparkdev amazing job. I left you a couple of suggestions, let me know what you think 😁

**Comment by jsparkdev** [comment]
thank you!
All the problems you mentioned have been resolved.

**Comment by dreyfus92** [comment]
Amazing, let's merge this 🫡

**Review by dreyfus92 — APPROVED** [review]

**Pull request merged**